applications/matter_bridge/CMakeLists.txt

```diff
@@ -23,44 +23,37 @@ find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
 
 project(matter-bridge)
 
-set(COMMON_ROOT ${ZEPHYR_NRF_MODULE_DIR}/samples/matter/common)
-set(NLIO_ROOT ${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/third_party/nlio/repo)
+# Enable GNU STD support and initialize the Matter Data Model.
 include(${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/config/nrfconnect/app/enable-gnu-std.cmake)
 include(${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/src/app/chip_data_model.cmake)
 
 # NORDIC SDK APP START
+
+# Include all source files that are located in the Matter common directory.
+include(${ZEPHYR_NRF_MODULE_DIR}/samples/matter/common/cmake/source_common.cmake)
+
 target_include_directories(app PRIVATE
     src
     src/bridged_device_types
-    ${COMMON_ROOT}/src
-    ${COMMON_ROOT}/src/bridge/
-    ${COMMON_ROOT}/src/bridge/util
-    ${NLIO_ROOT}/include
-    ${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/zzz_generated/app-common
+    ${MATTER_COMMONS_SRC_DIR}/bridge/
```

Comment:
nit: it may not be easy for a reader to know where `MATTER_COMMONS_SRC_DIR` points to without looking into `source_common.cmake`. To keep this file more self-contained, I would recommend using `${ZEPHYR_NRF_MODULE_DIR}/samples/matter/common`, which is longer but more explicit.